Calibration/LumiAlCaRecoProducers/python/alcaRawPCCProducer_cfi.py

```diff
@@ -1,15 +1,17 @@
 import FWCore.ParameterSet.Config as cms
 
+#note this default parameters are used to process the Random stream
 rawPCCProd = cms.EDProducer("RawPCCProducer",
     RawPCCProducerParameters = cms.PSet(
-        #Mod factor to count lumi and the string to specify output
-        inputPccLabel = cms.string("alcaPCCProducerRandom"),
+        inputPccLabel = cms.string("alcaPCCIntegratorRandom"),
         ProdInst = cms.string("alcaPCCRandom"),
-        resetEveryNLumi = cms.untracked.int32(1),
+        OutputValue = cms.untracked.string("Totals"),
+        ApplyCorrections = cms.untracked.bool(False),
         outputProductName = cms.untracked.string("rawPCCRandom"),
-        modVeto=cms.vint32()
+        modVeto=cms.vint32(),
+        saveCSVFile=cms.untracked.bool(False)
```

Comment:
in the .cc code please unify the style, so that is either `InputPccLabel` and `OutputProductName` and `ModVeto` and `SaveCSVFile` or it's `ProdInst` and `applyCorrections and `outputValue`

Reply:
I looked at the .cc code and the c++ variables all seem to follow the consistent lowercase. Its the python variables that need to be fixed (which also appear as strings in the cc), but I want to leave that for a future PR as quite a bit of clean up needs to be done in the python.